uefi/src/lib.rs

```diff
@@ -147,6 +153,16 @@
 //! only unfold their potential when you invoke `uefi::helpers::init` as soon
 //! as possible in your application.
 //!
+//! ### Recommended Default Features
+//!
+//! In typical use-cases, the following features are useful for you:
+//! - Building a UEFI image:
+//!   - Recommended: `alloc`, `global_allocator`, `logger`, `panic_handler`
+//!   - Optional   : `log-debugcon`, `unstable`, `qemu`
```

Comment:
When rendered by rustdoc, this text won't be monospace, so the extra spaces before `:` won't make the text line up anyway.

```suggestion
//!   - Optional: `log-debugcon`, `unstable`, `qemu`
```